spec/app/ics-721-nft-transfer/README.md

```diff
@@ -212,11 +212,13 @@ function onChanOpenInit(
   channelIdentifier: Identifier,
   counterpartyPortIdentifier: Identifier,
   counterpartyChannelIdentifier: Identifier,
-  version: string) {
+  version: string) (version: string, err: Error) {
   // only unordered channels allowed
   abortTransactionUnless(order === UNORDERED)
   // assert that version is "ics721-1"
-  abortTransactionUnless(version === "ics721-1")
+  // or relayer passed in empty version
+  abortTransactionUnless(version === "ics721-1" || version === "")
```

Comment:
We should communicate with the team implementing the NFT transfer app so that they take into account this change (if they don't already), right?